helpers/genRandomCodes.js

```diff
@@ -0,0 +1,59 @@
+const alpha = [
```

Comment:
Could we maybe use something like nanoid instead to generate the codes?

Reply:
Two reasons we aren't:
1) I didn't know about nanoid and for some reason didn't think to look for libraries that did this.
2) Our use-case is really simple and isn't being used in a high-security situation, so I think this is good, at least for now.
I did check out nanoid, but it seemed overly complicated for what we need. If we need something more secure in the future, I will look at it again and compare it to similar libraries.